app/components/account/TokenExtensionsCard.tsx

```diff
@@ -0,0 +1,283 @@
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('non-transferable-token')),
+                name: "Non-Transferable Token's Account",
+                status: 'active',
+            };
+            break;
+        }
+        case 'cpiGuard': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('cpi-guard')),
+                name: 'CPI Guard',
+                status: 'active',
+            };
+            break;
+        }
+        case 'permanentDelegate': {
+            return {
+                description:
+                    'Delegates permanent authority to a specific address that can transfer or burn tokens from any account holding this token, providing centralized administrative control over the token ecosystem.',
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('permanent-delegate')),
+                name: 'Permanent Delegate',
+                status: 'active',
+                tooltip:
+                    'Delegates permanent authority to a specific address that can transfer or burn tokens from any account holding this token, providing centralized administrative control over the token ecosystem.',
+            };
+            break;
+        }
+        case 'transferHook': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('transfer-hook')),
+                name: 'Transfer Hook',
+                status: 'active',
+            };
+            break;
+        }
+        case 'transferHookAccount': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('transfer-hook')),
+                name: "Transfer Hook's Account",
+                status: 'active',
+            };
+            break;
+        }
+        case 'metadataPointer': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('token-extensions-metadata')),
+                name: 'Metadata Pointer',
+                status: 'active',
+            };
+            break;
+        }
+        case 'groupPointer': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('group-member')),
+                name: 'Group Pointer',
+                status: 'active',
+            };
+            break;
+        }
+        case 'groupMemberPointer': {
+            return {
+                externalLinks: populateExternalLinks(populateSolanaDevelopersLink('group-member')),
+                name: 'Group Member Pointer',
+                status: 'active',
+            };
+            break;
+        }
+        case 'confidentialTransferAccount': {
+            return {
+                externalLinks: populateExternalLinks('https://spl.solana.com/confidential-token/quickstart'),
+                name: "Confidential Transfer's Account",
```

Comment:
description: "Token amount is only able to be unencrypted by the token holder or auditor key"

Reply:
fixed